scripts/windows-2019-provision.ps1

```diff
@@ -215,3 +250,23 @@ Select-Object -Property DeviceID, DriveType, VolumeName,
 
 Write-Host "Patch(s) installed"
 Get-HotFix | Format-Table -Property HotFixID, Description, InstalledOn
+
+Write-Host "== Sanity Check of installed tools"
+& docker -v ## Client only
+
+& "$baseDir\git\cmd\git.exe" --version
+& "$baseDir\jdk-8\bin\java.exe" -version
+& "$baseDir\jdk-11\bin\java.exe" -version
+& "$baseDir\jdk-17\bin\java.exe" -version
+& "$baseDir\container-structure-test.exe" version
+& "$baseDir\hadolint.exe" --version
+& "$baseDir\docker-compose.exe" --version
+& "$baseDir\git\mingw64\bin\git-lfs.exe" --version
+& "$baseDir\jx-release-version" --version
+& "$baseDir\jq.exe" --version
+& "C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin\az.cmd" version

```

Comment:
Same question, is this folder included in the $PATH?
Or should consumers of the tool know where they are stored?

Reply:
Same answer as the previous (good) question for the PATH.

In the case of the `azure-cli`, it's the MSI installation that decides of this default installation.
Did not check if I could customize the installation path: this "quick and dirty" absolute path is enough for the goal of the test: WDYT?